app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,55 @@
+# user_reviews GET    /users/:user_id/reviews(.:format)                reviews#index
+#              POST   /users/:user_id/reviews(.:format)                reviews#create
+# new_user_review GET    /users/:user_id/reviews/new(.:format)            reviews#new
+# edit_user_review GET    /users/:user_id/reviews/:id/edit(.:format)       reviews#edit
+#  user_review GET    /users/:user_id/reviews/:id(.:format)            reviews#show
+#              PATCH  /users/:user_id/reviews/:id(.:format)            reviews#update
+#              PUT    /users/:user_id/reviews/:id(.:format)            reviews#update
+#              DELETE /users/:user_id/reviews/:id(.:format)            reviews#destroy
+#
+#
+#              product_reviews GET    /products/:product_id/reviews(.:format)          reviews#index
+#                 POST   /products/:product_id/reviews(.:format)          reviews#create
+# new_product_review GET    /products/:product_id/reviews/new(.:format)      reviews#new
+# edit_product_review GET    /products/:product_id/reviews/:id/edit(.:format) reviews#edit
+#  product_review GET    /products/:product_id/reviews/:id(.:format)      reviews#show
+#                 PATCH  /products/:product_id/reviews/:id(.:format)      reviews#update
+#                 PUT    /products/:product_id/reviews/:id(.:format)      reviews#update
+#                 DELETE /products/:product_id/reviews/:id(.:format)      reviews#destroy
```

Comment:
Remember to delete comments like this before you commit, or if you think a teammate will need it for an intermediary commit, be sure to do a once-over of your code before your final pull request to remove these sorts of comments. :)